apps/web/src/hooks/usePermit2.ts

```diff
@@ -36,6 +36,15 @@ type UsePermit2ReturnType = Permit2HookState & Permit2HookCallback
 export const usePermit2 = (
   amount: CurrencyAmount<Token> | undefined,
   spender: Address | undefined,
+  options: {
+    /**
+     * Use paymaster if available.
+     * Enable only if Gas Token Selector is present on the interface.
+     */
+    enablePaymaster?: boolean
+  } = {
+    enablePaymaster: false,
+  },
```

Comment:
```suggestion
  {
      enablePaymaster = false,
    }: {
    /**
     * Use paymaster if available.
     * Enable only if Gas Token Selector is present on the interface.
     */
    enablePaymaster?: boolean
  } = {},
```

Reply:
please update to the same as what we did with `apps/web/src/hooks/useApproveCallback.ts`